docs/pages/blog/mui-x-v7.md

```diff
@@ -71,6 +71,10 @@ If you're using Material UI's theming system, the Data Grid will adapt to your
 Without it, you may need to manually set an appropriate background color to maintain your design.
 :::
 
+:::success
+Our teammate Romain has shared comprehensive insights into the optimization techniques in his blog post [Optimizing JavaScript](https://romgrk.com/posts/optimizing-javascript/).
```

Comment:
```suggestion
Our teammate [Romain](https://github.com/romgrk) wrote about performance-related optimization techniques, such as the abovementioned, in his blog post [Optimizing JavaScript](https://romgrk.com/posts/optimizing-javascript/).
```

A bit more context here + tieing it back to the blog post's content — cc @samuelsycamore for 👀

Reply:
I was pulling from the scrolling performance optimization/improvement on the Data Grid, where the callout is placed, to sort of tie it back to that (and thus kinda justify why we're linking this additional resource).